python/doc/examples/meta_modeling/polynomial_chaos_metamodel/plot_chaos_logNormal.py

```diff
@@ -0,0 +1,264 @@
+"""
+Pitfalls in  polynomial chaos expansion due to the input distribution
+"""
+
```

Comment:
try renaming without uppercase

Reply:
May you give me the command to prevent a mess in Github???